linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml

```diff
@@ -21,8 +21,6 @@ rationale: |-
 
 severity: unknown
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Containers could have chrony too. Reverting this change.

Reply:
ditto (see `chronyd_or_chronyc_network`)